### PR TITLE
Added bulk group member addition support

### PR DESCRIPTION
**Pull request by AKrasheninnikov**
With this new class **BulkAddGroupMemberRequest** you can add multiple users to a group by providing a list of emails. The result will contain arrays of Success and Failed results. Success will report detail on successfully added members and Failed will provide which email matches which error message.

The Server must provide API v. 2.1 in addition to 2.0.

### CODE REVIEW

**Comment by AKrasheninnikov** [comment]
Hmm... Package installed, but assembly cannot be loaded. Something to fix in Appveyor?

**Comment by renber** [comment]
Do you need features from Newtonsoft.Json Version 12 which is not provided in 8?
If we update Newonsoft.Json we would also enforce this newer version on all projects using the library, which could prove problematic (There are projects using this library which cannot use the newer version (e.g. Windows Phone Clients) which would then be locked out. I also have encountered problems with this version in Xamarin projects.)
Therefore, I would suggest to not update the dependency. Can you please update the pull request accordingly?

File: SeafClient/SeafClient.csproj [hunk 1]
```diff
@@ -132,9 +133,8 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
Comment (AKrasheninnikov) [inline SeafClient/SeafClient.csproj]: revert to v8

**Comment by AKrasheninnikov** [comment]
Can you validate the changes?

**Comment by renber** [comment]
You accidently deleted the packages.config file. Please restore the old one (which declares newtonsoft.json v.8.03) so that nuget's restore can download the package.

**Comment by AKrasheninnikov** [comment]
Yep. Just wanted to remove the file from committed changes. Not as easy to recover. Some issues with GitHub authentication won't let me publish the reverting commit.

**Comment by AKrasheninnikov** [comment]
Can you help? Can you only pull the one new .cs and include it in your version of the build?

**Comment by renber** [comment]
I merged your PR and reverted packages.config to the original version.
Can you please create another PR containing the corresponding unit tests for this request in the test project. I will then create a new Nuget release.

**Pull request merged**